### PR TITLE
Port System.Security.Cryptography.X509Certificates triple slash to docs

### PR DESCRIPTION
**Pull request by carlossanlop**
Ported with DocsPortingTool.

@gewarren targeting the net6-rc1 branch.

### CODE REVIEW

**Comment by gewarren** [comment]
Docs Build status updates of commit _[66cebdd](https://github.com/carlossanlop/dotnet-api-docs/commits/66cebdd620d00a6431c5eef7f961e1c3a9d03017)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[xml/System.Security.Cryptography.X509Certificates/PublicKey.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp.x509/xml/System.Security.Cryptography.X509Certificates/PublicKey.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.publickey?view=net-5.0&branch=pr-en-us-7095) |
[xml/System.Security.Cryptography.X509Certificates/X509Certificate2.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp.x509/xml/System.Security.Cryptography.X509Certificates/X509Certificate2.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.x509certificate2?view=net-5.0&branch=pr-en-us-7095) |
[xml/System.Security.Cryptography.X509Certificates/X509Certificate2Collection.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp.x509/xml/System.Security.Cryptography.X509Certificates/X509Certificate2Collection.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.x509certificate2collection?view=net-5.0&branch=pr-en-us-7095) |
[xml/System.Security.Cryptography.X509Certificates/X509Certificate2Enumerator.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp.x509/xml/System.Security.Cryptography.X509Certificates/X509Certificate2Enumerator.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.x509certificate2enumerator?view=net-5.0&branch=pr-en-us-7095) |
[xml/System.Security.Cryptography.X509Certificates/X509ChainElementCollection.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp.x509/xml/System.Security.Cryptography.X509Certificates/X509ChainElementCollection.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.x509chainelementcollection?view=net-5.0&branch=pr-en-us-7095) |
[xml/System.Security.Cryptography.X509Certificates/X509ChainElementEnumerator.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp.x509/xml/System.Security.Cryptography.X509Certificates/X509ChainElementEnumerator.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.x509chainelementenumerator?view=net-5.0&branch=pr-en-us-7095) |
[xml/System.Security.Cryptography.X509Certificates/X509ExtensionCollection.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp.x509/xml/System.Security.Cryptography.X509Certificates/X509ExtensionCollection.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.x509extensioncollection?view=net-5.0&branch=pr-en-us-7095) |
[xml/System.Security.Cryptography.X509Certificates/X509ExtensionEnumerator.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp.x509/xml/System.Security.Cryptography.X509Certificates/X509ExtensionEnumerator.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.x509extensionenumerator?view=net-5.0&branch=pr-en-us-7095) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C3%5C500c749d-de78-3d4a-00e4-269d599bf9ec%5CPullRequest%5C202109030654423544-7095%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=0xexCzqqbkLksN2F1WpbEMvh5Ro7ReCBG5dTfiGKH6A%3D&st=2021-09-03T07%3A49%3A50Z&se=2021-10-04T07%3A54%3A50Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/500c749d-de78-3d4a-00e4-269d599bf9ec?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: xml/System.Security.Cryptography.X509Certificates/PublicKey.xml [hunk 1]
```diff
@@ -526,10 +534,11 @@
         <Parameter Name="bytesWritten" Type="System.Int32" RefType="out" Index="1" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="destination">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="destination">The byte span to receive the X.509 SubjectPublicKeyInfo data.</param>
+        <param name="bytesWritten">When this method returns, contains a value that indicates the number of bytes written to <paramref name="destination" />. This parameter is treated as uninitialized.</param>
```
Comment (gewarren) [inline xml/System.Security.Cryptography.X509Certificates/PublicKey.xml:538]:
```suggestion
        <param name="bytesWritten">When this method returns, contains the number of bytes written to <paramref name="destination" />. This parameter is treated as uninitialized.</param>
```

File: xml/System.Security.Cryptography.X509Certificates/X509Certificate2.xml [hunk 2]
```diff
@@ -1237,10 +1243,32 @@ The password specified for the private key is incorrect.</exception>
...
+        <param name="certPem">The text of the PEM-encoded X509 certificate.</param>
+        <summary>Creates a new X509 certificate from the contents of an RFC 7468 PEM-encoded certificate.</summary>
+        <returns>A new X509 certificate.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+
+            This loads the first well-formed PEM found with a CERTIFICATE label.
```
Comment (gewarren) [inline xml/System.Security.Cryptography.X509Certificates/X509Certificate2.xml:1255]:
```suggestion
This loads the first well-formed PEM found with a CERTIFICATE label.
```

File: xml/System.Security.Cryptography.X509Certificates/X509Certificate2.xml [hunk 3]
```diff
@@ -1237,10 +1243,32 @@ The password specified for the private key is incorrect.</exception>
...
+            
+
+
+            For PEM-encoded certificates with a private key, use
```
Comment (gewarren) [inline xml/System.Security.Cryptography.X509Certificates/X509Certificate2.xml:1259]:
```suggestion
For PEM-encoded certificates with a private key, use
```

File: xml/System.Security.Cryptography.X509Certificates/X509Certificate2.xml [hunk 4]
```diff
@@ -1237,10 +1243,32 @@ The password specified for the private key is incorrect.</exception>
...
+            <xref:System.Security.Cryptography.X509Certificates.X509Certificate2.CreateFromPem(System.ReadOnlySpan{System.Char},System.ReadOnlySpan{System.Char})>.
+            
+
+
+            For PEM-encoded certificates in a file, use <xref:System.Security.Cryptography.X509Certificates.X509Certificate2.#ctor(System.String)>.
```
Comment (gewarren) [inline xml/System.Security.Cryptography.X509Certificates/X509Certificate2.xml:1264]:
```suggestion
For PEM-encoded certificates in a file, use <xref:System.Security.Cryptography.X509Certificates.X509Certificate2.#ctor(System.String)>.
```

**Review by gewarren — APPROVED** [review]

**Review by bartonjs — APPROVED** [review]

**Pull request merged**